modules/data.land/R/soilgrids_soc_extraction.R

```diff
@@ -65,10 +65,10 @@ soilgrids_soilC_extract <- function (site_info, outdir=NULL, verbose=TRUE) {
   }
   
   # prepare site info for extraction
-  internal_site_info <- data.frame(site_info$site_id, site_info$site_name, site_info$lat,site_info$lon)
+  internal_site_info <- site_info[, c("site_id", "site_name", "lat", "lon")]
```

Comment:
Mostly to simplify the column names when working with them below, but has a nice side benefit: If the user passes extra columns in `site_info` that this function doesn't need, the code will now ignore them rather than complain.

Reply:
I should add the complaint about extra columns didn't happen here, but in the `colnames<-` on now-removed line 108.